File: core/src/main/java/com/rudderstack/android/sdk/core/RudderOSInfo.java
package com.rudderstack.android.sdk.core;

import android.os.Build;

import com.google.gson.annotations.SerializedName;

class RudderOSInfo {
    @SerializedName("name")
    private final String name = "Android";
    @SerializedName("version")
    private final String version = Build.VERSION.RELEASE;
}

File: android/src/main/java/com/rudderstack/android/models/RudderOSInfo.kt
package com.rudderstack.android.models

import com.google.gson.annotations.SerializedName

class RudderOSInfo(
    @SerializedName("name")
    private val name: String = "Android",

    @SerializedName("version") // = Build.VERSION.RELEASE
    private val version: String? = null,
)
